en/payloads/README.md

```diff
@@ -24,55 +24,65 @@ PX4 supports the following set of MAVLink commands/mission items for cameras tha
 * [MAV_CMD_DO_TRIGGER_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_TRIGGER_CONTROL) - start/stop capturing (using distance or time, as defined using above messages).
 
 The following topics show how to *connect* your camera configure PX4:
-* [Camera triggering](../peripherals/camera.md) from flight controller PWM or GPIO outputs, or via MAVLink.
+
+* [Camera triggering](../peripherals/camera.md) from flight controller PWM or GPIO outputs, or via MAVLink. 
 * [Camera timing feedback](../peripherals/camera.md#camera-capture) via hotshoe input.
 
 
 ## Cargo Drones ("Actuator" Payloads)
 
 Cargo drones commonly use servos/actuators to trigger cargo release, control winches, etc.
-PX4 supports servo and GPIO triggering via both RC and MAVLink commands.
+PX4 supports actuator triggering via both RC and MAVLink commands.
 
-### RC Triggering
+### Payload Outputs: Control Allocation
+
+If control allocation is enabled ([SYS_CTRL_ALLOC=1](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC)) you can specify up to 6 outputs to be controlled from either RC channels or MAVLink.
 
-You can map up to three RC channels to control servos/actuators attached to the flight controller using the parameters [RC_MAP_AUX1](../advanced_config/parameter_reference.md#RC_MAP_AUX1) to [RC_MAP_AUX3](../advanced_config/parameter_reference.md#RC_MAP_AUX3).
+:::note
+Control allocation is supported from PX4 v1.13 but disabled by default.
+:::
 
-The RC channels are *usually* mapped to the `AUX1`, `AUX2`, `AUX3` outputs of your flight controller (using a [mixer file](../concept/mixing.md) defined in your airfame).
+This is done in the [Actuators](../config/actuators.md#actuator-outputs) configuration screen by assigning the functions `Offboard Actuator Set 1` to `Offboard Actuator Set 6` functions to any desired payload outputs (in the [actuator outputs](../config/actuators.md#actuator-outputs) section).
```

Comment:
Note that the ones for RC are the `RC_AUX x` functions

Reply:
Not sure how this applies to this statement. Will discuss below.